src/components/FilterListItem/FilterListItem.jsx

```diff
@@ -21,15 +21,15 @@ class FilterListItem extends React.Component {
               className="button is-link"
               title="Edit filter"
             >
-              <span className="octicon octicon-pencil"></span>
+              ✏️
             </button>
             <button
               onClick={() => this.props.delete(this.props.filter)}
               type="button"
               className="button is-link"
               title="Delete filter"
             >
-              <span className="octicon octicon-trashcan"></span>
+              ❌
```

Comment:
Haha okay clearly not that emoji if it's not part of the standard set. I'm in El Capitan and that shows up as a trash can for me:

![screen shot 2016-09-21 at 6 21 48 pm](https://cloud.githubusercontent.com/assets/82317/18732384/51fe9782-8028-11e6-8c4e-76496e150ef3.png)

Keep this X for now!